src/tests/processor/tests.py

```diff
@@ -0,0 +1,190 @@
+import unittest
+from unittest.mock import patch
+from urllib.parse import unquote
+
+from processor.logic import (
+    check_wikipedia_event,
+    get_wiki_page_title,
+    get_wikipedia_references,
+    get_language_from_wiki_url,
+    get_non_wikipedia_references,
+    get_wikimedia_call_info,
+    is_wikipedia_url,
+)
+
+
+class WikipediaTestCase(unittest.TestCase):
+
+    base_netloc = 'sv.wikipedia.org'
+    base_url = f'https://{base_netloc}/wiki'
+
+    page_title = 'Festival_Romanistica'  # Page titles
+    page_title_encoded = 'Fanny_Ambj%C3%B6rnsson'
+
+    class MockUri(object):
+        def __init__(self, raw):
+            self.raw = raw
+
+    class MockEvent(object):
```

Comment:
Cool, declaring classes used for testing here makes a lot more sense than doing so in the setUpTestData method like I've done in Central Station.